app/workers/delete_account_hierarchy_worker.rb

```diff
@@ -14,7 +14,7 @@ def batch_description
     id = account.id
     org_name = account.org_name
     if account.provider?
-      "Deleting provider [##{id}] #{org_name} - #{account.admin_domain}"
+      "Deleting provider [##{id}] #{org_name} - #{account.external_admin_domain}"
```

Comment:
Not sure exactly where this is used, but assuming that it's for internal logs, I would suggest `internal_admin_domain` here.

Reply:
I saw the string and thought it would be an alert or something but I agree if it's a log it should be internal.